### DISCUSSION TITLE
Email verification isn't necessary to be able to have a working account

### DISCUSSION THREAD

**Discussion by vincentsijben** [processing/p5.js-web-editor#2669] [General]
### What is your operating system?

Mac OS

### Web browser and version

Chrome 113.0.5672.92

### Actual Behavior

When I sign up using the email option, I receive an email with a verification link. I didn't click that link on purpose, and after a week I can still login with my signed up credentials. 

What's interesting though is that I like this idea of very-easy account creation for schools. My students have upcoming projects with elementary schools and all those kids will be using the online p5js editor. But they need to save their work, and don't have a GitHub or Google account (they do have a Microsoft 365 email address though). So with this bug, my students are able to gather all the kids' email addresses and just create accounts fór them, with the same (or not) passwords, to be able to quickly login as well and help remotely when there are problems. 

### Expected Behavior

So maybe, it's an option for people who sign up with email addresses not having to verify them and perhaps delete those accounts that didn't login or saved any work after x months?

### Steps to reproduce

* Sign up using the email address option.
* Don't click the verification link
* Login into the webeditor

**Comment by welcome** [comment]
Welcome! 👋 Thanks for opening your first issue here! And to ensure the community is able to respond to your issue, be sure to follow the issue template if you haven't already.
